server/src/test/java/io/pravega/schemaregistry/storage/impl/group/GroupPravegaTest.java

```diff
@@ -0,0 +1,757 @@
+        Etag etag = pravegaKVGroups.getGroup(null, groupName).join().getCurrentEtag().join();
+        pravegaKVGroups.getGroup(null, groupName).join().addSchema(schemaInfo, groupProperties, etag).join();
+        schemaWithVersionList = pravegaKVGroups.getGroup(null, groupName).join().getLatestSchemas().join();
+        assertEquals(1, schemaWithVersionList.size());
+        assertEquals("anygroup", schemaWithVersionList.get(0).getSchemaInfo().getType());
+        // two schemas
+        etag = pravegaKVGroups.getGroup(null, groupName).join().getCurrentEtag().join();
+        pravegaKVGroups.getGroup(null, groupName).join().addSchema(schemaInfo1, groupProperties, etag).join();
+        schemaWithVersionList = pravegaKVGroups.getGroup(null, groupName).join().getLatestSchemas().join();
+        assertEquals(2, schemaWithVersionList.size());
+        assertEquals("anygroup", schemaWithVersionList.get(0).getSchemaInfo().getType());
+        assertEquals("anygroup1", schemaWithVersionList.get(1).getSchemaInfo().getType());
+    }
+
+    @Test
+    public void testAddSchema() {
+        GroupProperties groupProperties = GroupProperties.builder().allowMultipleTypes(Boolean.FALSE).properties(
+                ImmutableMap.<String, String>builder().build()).serializationFormat(
+                SerializationFormat.custom("custom1")).compatibility(
+                Compatibility.forward()).build();
+        pravegaKVGroups.addNewGroup(null, groupName, groupProperties).join();
+        pravegaKVGroups.getGroup(null, groupName).join().create(SerializationFormat.Custom,
+                ImmutableMap.of(),
+                Boolean.TRUE, Compatibility.backward()).join();
+        byte[] schemaData = new byte[0];
+        SchemaInfo schemaInfo = new SchemaInfo("anygroup", SerializationFormat.Custom, ByteBuffer.wrap(schemaData),
+                ImmutableMap.of());
+        Etag etag = pravegaKVGroups.getGroup(null, groupName).join().getCurrentEtag().join();
+        pravegaKVGroups.getGroup(null, groupName).join().addSchema(schemaInfo, groupProperties, etag).join();
+        GroupsValue gv = tableStore.getEntry(GROUPS, new NamespaceAndGroup(null, groupName).toBytes(),
+                GroupsValue::fromBytes).join().getRecord();
+        // one schema
+        //versionInfo key
+        TableRecords.SchemaRecord schemaRecord = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()),
+                new TableKeySerializer().toBytes(new TableRecords.VersionKey(0)),
+                x -> TableRecords.fromBytes(TableRecords.VersionKey.class, x,
+                        TableRecords.SchemaRecord.class)).join().getRecord();
+        assertEquals("anygroup", schemaRecord.getSchemaInfo().getType());
+        //schemaInfo key
+        TableRecords.SchemaVersionList schemaVersionList = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()), new TableKeySerializer().toBytes(
```

Comment:
same thing here.. use the constant from PravegaKVGroup.java (make it package visible)